clients/web/src/common/api.js

```diff
@@ -39,64 +47,114 @@ function request(endpoint, { body, method, ...customConfig } = {}) {
   return fetch(url, config)
 }
 
-export async function advanceAssessment(assessment) {
-  const { id, status, targets, practices, questions, answers } = assessment
-  const currentStepIndex = VALID_ASSESSMENT_STEPS.indexOf(status)
-  if (
-    currentStepIndex >= 0 &&
-    currentStepIndex < VALID_ASSESSMENT_STEPS.length - 1
-  ) {
-    const nextStep = VALID_ASSESSMENT_STEPS[currentStepIndex + 1]
-    const response = await request(`/assessments/${id}`, {
-      method: 'PATCH',
-      body: { status: nextStep },
-    })
-    if (!response.ok) throw new APIError(response.status)
-    const data = await response.json()
-    return new Assessment({
-      ...data.assessment,
-      targets,
-      practices,
-      questions,
-      answers,
-    })
-  }
-  return assessment
-}
-
-export async function createAssessment(payload) {
-  const response = await request('/assessments', { body: payload })
+export async function createAssessment(organizationId, payload) {
+  const response = await request(`/${organizationId}/sample/`, {
+    body: payload,
+  })
   if (!response.ok) throw new APIError(response.status)
-  const { assessment } = await response.json()
-  return new Assessment(assessment)
+  const { slug, created_at } = await response.json()
+  return new Assessment({ id: slug, created: created_at })
 }
 
+// TODO: Review
 export async function getAnswers(organizationId, assessmentId) {
```

Comment:
Based on the `getAssessment` implementation below, is this code still necessary?

Reply:
Probably not. I will check this method and `getQuestions` tomorrow. Most likely, I'll remove them.

Reply:
I ended up removing this method and `getQuestions`.